analyzers/golang/analyze.go

```diff
@@ -38,10 +38,17 @@ func (a *Analyzer) Analyze() (graph.Deps, error) {
 		if a.Options.LockfilePath == "" {
```

Comment:
I think it actually makes sense to specify default manifest and lockfile paths for each strategy (using the package manager's default conventions). I'm okay with you adding that into this PR or a follow-up.

Reply:
Agreed, judging by the line you selected are you also suggesting we remove the empty string check becuase we will no longer support custom Lockfile paths(in which case it cannot be empty)?